src/saml.ts

```diff
@@ -265,6 +265,14 @@ class SAML {
       request["samlp:AuthnRequest"]["@AssertionConsumerServiceURL"] = this.getCallbackUrl(host);
     }
 
+    const samlExtensions = this.options.samlExtensions;
+    if (samlExtensions != null) {
```

Comment:
If this is a `number` or `string` it will cause problems, especially if we try to ...spread it. We might as well do a `typeof` check here.

Reply:
agreed